modules/web/src/app/dynamic/enterprise/cluster-backups/list/backup-storage-location/add-dialog/component.ts

```diff
@@ -192,7 +193,7 @@ export class AddBackupStorageLocationDialogComponent implements OnInit, OnDestro
           prefix: this.form.get(Controls.Prefix).value,
           caCert: this.form.get(Controls.CaCert).value,
         },
-        backupSyncPeriod: this.form.get(Controls.BackupSyncPeriod).value,
+        backupSyncPeriod: this.form.get(Controls.BackupSyncPeriod).value || null,
```

Comment:
Since `0` is a valid value, `||` would set the value to `null`.

Reply:
Good Catch @Waseem826